### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by gmokki**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@
 - Expose wakeup via REST-API
 - Update database indices to match workflow instance polling code
 - Add new configuration option `nflow.db.disable_batch_updates`
+- Workarounds for Galera oddities to make nflow more stable
```
Comment (efonsell) [inline CHANGELOG.md]:
```suggestion
- Workarounds for Galera oddities to make nFlow more stable when Galera is used
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -28,7 +29,7 @@
   - Update database indices to match current workflow instance polling code.
   - Create indices for foreign keys in MS SQL database.
   - Fix create database scripts to work with empty database.
-  - Add `nflow.db.disable_batch_updates` (default `false`) configuration parameter to make it possible to force use of multiple updates even if batch updates are supported by the database.
+  - Add `nflow.db.disable_batch_updates` (default `false`) configuration parameter to make it possible to force use of multiple updates even if batch updates are supported by the database. This is a needed forkaround on some Galera systems that cannot handle batches correctly.
```
Comment (efonsell) [inline CHANGELOG.md]:
```suggestion
  - Add `nflow.db.disable_batch_updates` (default `false`) configuration parameter to make it possible to force use of multiple updates even if batch updates are supported by the database. This is needed on some Galera systems that cannot handle batch updates correctly.
```

File: CHANGELOG.md (hunk 1 above)
Comment (ttiurani) [reply CHANGELOG.md]: I don't think we did anything except that above `nflow.db.disable_batch_updates` actually. If not documenting in Wiki counts?

File: CHANGELOG.md [hunk 3]
```diff
@@ -66,6 +67,7 @@
 - Improve error logging in WorkflowStateProcessor.
 - Replace FindBugs with SpotBugs.
 - Drop index from main table that was only used for archiving purposes.
+- Increase workflow and action identifiers in code to 64 bits for future proofing. The SQL table definitions stay the same for now, but no code changes are needed if database is altered to use bigint identifiers in large systems. The default 31 bits should be enough for 10 workflows per second with each having 20 steps for 8 years constant load.
```
Comment (ttiurani) [inline CHANGELOG.md]: Here the caveat is that multi-master databases jump over a lot of indices. In our galera the gap is 3, so 1, 4, 7, 10...

**Comment by efonsell** [comment]
When this is merged, please update https://github.com/NitorCreations/nflow/releases/tag/6.0.0 also

File: CHANGELOG.md [hunk 4]
```diff
@@ -66,6 +66,7 @@
...
```
Comment (efonsell) [inline CHANGELOG.md]:
```suggestion
- Increase workflow and action identifiers in code to 64 bits for future proofing. Database schema is not changed for now, but the columns can be altered later (to bigints) if needed.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -66,6 +66,7 @@
 - Improve error logging in WorkflowStateProcessor.
 - Replace FindBugs with SpotBugs.
 - Drop index from main table that was only used for archiving purposes.
+- Increase workflow and action identifiers in code to 64 bits for future proofing. Database schema is not changed for now, but the columns can be altered later (to bigints) if needed.
```
Comment (efonsell) [inline CHANGELOG.md]:
```suggestion
- Increase workflow instance and action identifiers in code to 64 bits for future proofing. Database schema is not changed for now, but the columns can be altered later (to bigints) if needed.
```

**Pull request merged**